circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/event/CompositeTableReplicationListener.java

```diff
@@ -40,7 +41,7 @@ public void tableReplicationStart(EventTableReplication tableReplication, String
       try {
         listener.tableReplicationStart(tableReplication, eventId);
       } catch (Exception e) {
-        LOG.error("Listener '{}' threw exception on tableReplicationStart.", listener, e);
+        throw new CircusTrainException("Listener '" + listener + "' threw exception on tableReplicationStart.", e);
```

Comment:
Need a unit test for this code path.

Reply:
Not sure if this is a good idea, cause it means the other listeners aren't run anymore.

See the other listener messages, they all log and we should end of in a state of replication failure.